kagami/database/models/helper.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import String
+from sqlalchemy.orm import Mapped, mapped_column
+
+from .base import BaseModel
+
+
+class Helper(BaseModel):
+    __tablename__ = "helper"
+
+    name: Mapped[str] = mapped_column(primary_key=True)
+    content: Mapped[str] = mapped_column(String(256))
```

Comment:
Use Text Instead of String, See https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.TEXT